.github/workflows/homebrew-update-current.yml

```diff
@@ -0,0 +1,25 @@
+name: update_homebrew_current
+on:
+  release:
+    # edited is the event triggered when changing from a pre-release to a release on a published release
+    type: [edited]
```

Comment:
Will this run if we do other edits to the release e.g. edit to add extra notes to the release?

This documentation: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release Seems to indicate that `released` triggers when a pre-release is changed to a release, which might make more sense.